cmake/onnxruntime_providers_migraphx.cmake

```diff
@@ -21,8 +21,8 @@
   # Add search paths for default rocm installation
   list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hcc /opt/rocm/hip /opt/rocm $ENV{HIP_PATH})
 
-  # Suppress the warning about the small capitals of the package name
-  cmake_policy(SET CMP0144 NEW)
+  # Suppress the warning about the small capitals of the package name - Enable when support to CMake 3.27.0 is used
```

Comment:
```suggestion
if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.27)
    cmake_policy(SET CMP0144 NEW)
endif()
```

Reply:
I'm just going to comment this out for now.